ansible_collections/arista/cvp/docs/modules/cv_container_v3.rst.md

```diff
@@ -15,6 +15,14 @@ containers with their parent, to create and delete containers on CVP
 side. Module also supports to configure configlets at container level.
 Returns number of created and/or deleted containers
 
+The module doesn't merge the existing container structure with the one
+specified in your playbook's topology; instead, it will try to remove
+unspecified containers from CVP. Existing containers are removed under
+two circumstances:
+
+-   The container has no confliglets attached to it.
+-   Strict apply_mode is set (see below).
```

Comment:
```suggestion
With the argument `apply_mode` set to `loose` the module will only add new containers.
When `apply_mode` is set to `strict` the module will try to remove
unspecified containers from CloudVision. This will fail if the container has configlets attached to it or devices are placed in the container.
```